torch/_dynamo/symbolic_convert.py

```diff
@@ -2204,6 +2214,7 @@ def symbolic_locals_contain_module_class(self):
     def RETURN_VALUE(self, inst):
         if (
             self.output.count_calls() == 0
```

Comment:
@voznesenskym this is where the current count_call() test I was thinking of lives